src/Maestro/Maestro.ContainerApp/Queues/BackgroundQueueListener.cs

```diff
@@ -71,18 +74,27 @@ private async Task ProcessItemAsync(BackgroundWorkItem item, CancellationToken c
         switch (item)
         {
             case StartSubscriptionUpdateWorkItem startSubscriptionUpdate:
-                _logger.LogInformation($"Processing { nameof(StartSubscriptionUpdateWorkItem) }: { startSubscriptionUpdate.SubscriptionId }");
```

Comment:
I think, we can have this log record to speed up debugging.

Reply:
The processor does that inside automatically